framework/source/class/qx/test/bom/webfonts/Manager.js

```diff
@@ -89,6 +69,30 @@ qx.Class.define("qx.test.bom.webfonts.Manager", {
       this.__nodesBefore = document.body.childNodes.length;
       this.__sheetsBefore = document.styleSheets.length;
       this.__manager = qx.bom.webfonts.Manager.getInstance();
+
+      this.__fontDefinitions = {
+        finelinerScript :
+        {
```

Comment:
Is __findRule called only once? If __fontDefinitions needs to be different for different instances, then typically the initialization would be done in the constructor.